src/main/java/org/mskcc/oncokb/model/Genomic.java

```diff
@@ -0,0 +1,283 @@
+package org.mskcc.oncokb.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Jing Su
+ */
+
+public class Genomic implements java.io.Serializable{
```

Comment:
We may need to abstract oncokbVariants out of Genomic. But we can ignore for now